### PR TITLE
Performs all CRUD operations.

### PR DESCRIPTION
**Pull request by asuluntas**
Hi Uma and Andrew,
I completed CRUD operations in the following files:
server/db/index.js 
server/index.js
Please let me know if there is something that I need to fix.
PS: I forgot to merge the previous PR. I merged it just before this PR so it looks there is no change.
Thanks,
Aysun

### CODE REVIEW

**Comment by uabrami** [comment]
Thanks Aysun, looks good! For your video and associated videos pull request, have you placed an index in postgres for name and url? Andrew suggested this to me, so I did this for the columns I am matching by, and my search time is very fast. If you haven't already indexed these columns, I suggest you do so and test the time again. It would be great to be able to include that post req if possible since it updates both the movie and associated videos.

Best,
Uma

File: generateRandomId.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const writer = csvWriter();
+const faker = require('faker');
+
+
+const generateId = () => {
+  writer.pipe(fs.createWriteStream('id.csv'));
+  for (let i = 0; i < 10000; i++) {
+    writer.write({
+      id: faker.random.number({'min': 1,'max': 10000})
+    });
```
Comment (ahchung23) [inline generateRandomId.js:12]: I think we may need to change the number of random Id's to 250 depending on what kind of feedback we get about the max limit of event types and why we're dropping data points

File: test.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      # x-my-service-auth: '987401838271002188298567'
+  payload:
+    path: "id.csv"
+    fields:
+      - "id"
+    skipheader: true
+scenarios:
+  - name: "GET requests"
+    weight: 19
+    flow:
```
Comment (ahchung23) [inline test.yml:22]: Nice weighting of scenarios Aysun! You could go even lower than 5% if you wanted to.

Do you think waiting 5 seconds after get request is realistic? Or do you feel we should be making it longer/shorter?

**Review by ahchung23 — COMMENTED** [review]
lgtm Aysun. I can't find any major issues!

I'm so sad you had to comment out the cache section! we should definitely implement that strategy later to decrease our server latency!

**Pull request merged**